src/components/Locationitem/LocationItem.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { describe, it, expect, vi } from "vitest";
+import { render, screen, fireEvent } from "@testing-library/react";
+import LocationItem from "./LocationItem";
+
+describe("<LocationItem />", () => {
+  it("renders the location name", () => {
+    render(<LocationItem text="New York" />);
+    expect(screen.getByText("New York")).toBeInTheDocument();
+  });
+
+  it("calls handleLocation when clicked", () => {
+    //creates a mock function
```

Comment:
don't need this comment right?

Reply:
It's for me to remember what each thing does 😬